Library/Koala/Koala.swift

```diff
@@ -2048,6 +2087,39 @@ private func properties(
     .withAllValuesFrom(loggedInUserProperties)
 }
 
+private func properties(
+  project: Project,
+  reward: Reward? = nil,
+  backing: Backing? = nil,
+  prefix: String = "project_"
+) -> [String: Any] {
+  var props: [String: Any] = [:]
+
+  props["name"] = project.name
+  props["pid"] = project.id
+  props["category"] = project.category.name
+  props["has_video"] = project.video != nil
+  props["location"] = project.location.name
+  props["country"] = project.country.countryCode
+
+  let now = MockDate().timeIntervalSince1970
```

Comment:
I see that on line 2069 we are using `MockDate()` too, but I think we should be using `Date()` instead. @justinswart Do you know the reason for this?

Reply:
Hmm well-spotted @Scollaco. I can't imagine why `MockDate()` was used in this file, I see it's a change I made in #67 to fix tests, but I think doing it in this file, which is application code and not unit test code, is incorrect.

The only reason we use `MockDate()` in tests is to give ourselves control over the movement of time. It also ensures that tests pass in environments that are in different timezones, etc.

So yes, I think both instances of `MockDate()` here should be `AppEnvironment.current.dateType.init()`. This means that in unit tests it will use `MockDate` and in the app it uses `Date`

This likely means that the value of `hours_remaining` in our tracking properties has always been incorrect 🤔 🤷‍♂